app/helpers/application_helper.rb

```diff
@@ -96,13 +95,13 @@ def format_conference_twitter(twitter)
   def links_to_conferences(conferences)
     conferences.map do |conference|
       text = conference.name
-      extra = [conference.location, format_conference_date(conference.starts_on, conference.ends_on)].reject(&:blank?).join(' – ')
-      text += " (#{extra})" unless extra.blank?
+      details = [conference.location, format_conference_date(conference.starts_on, conference.ends_on)].reject(&:blank?).join(' – ')
+      text += " (#{details})"
       link_to(text, conference)
```

Comment:
Well, if the condition is gone now, I'd suggest to make this all a bit smaller:
```ruby
def links_to_conferences(conferences)
  details = [conference.location, format_conference_date(conference.starts_on, conference.ends_on)].reject(&:blank?).join(' – ')
  link_to "#{conference.name} (#{details})", conference
end
```

But well, optional 😉